hassio/src/addon-view/documentation/hassio-addon-documentation-tab.ts

```diff
@@ -64,6 +64,39 @@ class HassioAddonDocumentationDashboard extends LitElement {
           padding: 8px;
           max-width: 1024px;
         }
+        ha-markdown {
+          display: block;
+          padding: 0 16px 16px;
+          -ms-user-select: initial;
+          -webkit-user-select: initial;
+          -moz-user-select: initial;
+        }
+        .markdown.no-header {
+          padding-top: 16px;
+        }
+        ha-markdown > *:first-child {
+          margin-top: 0;
+        }
+        ha-markdown > *:last-child {
+          margin-bottom: 0;
+        }
+        ha-markdown a {
+          color: var(--primary-color);
+        }
+        ha-markdown img {
+          max-width: 100%;
+        }
+        ha-markdown code,
+        pre {
```

Comment:
Is this default styling that we should extract out and be able to include in any component that renderrs markdown and wants to use this styling? 

Also, let's add some padding to the code block like GitHub does.

Reply:
I think I added it to all that uses it now, but it should probably be extracted yes.
I'll do that 👍